docker/docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+
+services:
+  clio:
+    image: ${CLIO_IMAGE}
+    build:
+      context: ..
+      dockerfile: docker/Dockerfile
+    network_mode: host
```

Comment:
Why network mode host for all containers? I think it is better to map ports. Also is there any point to expose cassandra to the host?

Reply:
What I really would prefer would be the user bridge network so one can just use DNS names where applicable. I'm not too keen on using the host network usually but I think there's a field in rippled's config that doesn't allow for a DNS name (one in Clio too I think.)

I think quickly using host network to be simpler but I'm not going to defend that position too vehemently if you think port-mapping is more appropriate.